.github/workflows/enable-pack-build-and-test.yaml

```diff
@@ -0,0 +1,115 @@
+name: Enable Pack Build and Test
+
+on:
+  # First day of the month at midnight.
+  schedule:
+    - cron: "0 0 1 * *"
```

Comment:
If we'll run the re-enable workflow once a month, there's a ~29 days for packs CI to expire.
With that, I think a daily run would be sufficient to cover all the `60d` expiration cases.